packages/react-components/src/components/VideoTile.tsx

```diff
@@ -172,23 +173,26 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
 
   const ids = useIdentifiers();
 
-  const isSpeakingStyles = isSpeaking
-    ? {
-        border: `0.25rem solid ${theme.palette.themePrimary}`
-      }
-    : {};
-
   return (
     <Ref innerRef={videoTileRef}>
       <Stack
         data-ui-id={ids.videoTile}
         className={mergeStyles(
           rootStyles,
-          isSpeakingStyles,
-          { background: theme.palette.neutralLighter, borderRadius: theme.effects.roundedCorner4 },
+          {
+            background: theme.palette.neutralLighter,
+            borderRadius: theme.effects.roundedCorner4
```

Comment:
nit: it would be nice to see this and the borderRadius below using the same const variable so its clear they're linked and if one is updated, the other is updated